src/System.Management.Automation/FormatAndOutput/common/TableWriter.cs

```diff
@@ -390,17 +383,36 @@ private string[] GenerateTableRow(string[] values, ReadOnlySpan<int> alignment,
             for (int row = 0; row < screenRows; row++)
             {
                 StringBuilder sb = new StringBuilder();
+
                 // for a given row, walk the columns
                 for (int col = 0; col < scArray.Length; col++)
                 {
+                    string value = scArray[col][row];
+
                     // if the column is the last column with content, we need to trim trailing whitespace, unless there is only one row
                     if (col == lastColWithContent[row] && screenRows > 1)
                     {
-                        sb.Append(scArray[col][row].TrimEnd());
+                        value = value.TrimEnd();
                     }
-                    else
+
+                    if (isHeader)
                     {
-                        sb.Append(scArray[col][row]);
+                        if (_si.columnInfo[col].HeaderMatchesProperty || !ExperimentalFeature.IsEnabled(ExperimentalFeature.PSCustomTableHeaderLabelDecoration))
```

Comment:
```
|| !ExperimentalFeature.IsEnabled(ExperimentalFeature.PSCustomTableHeaderLabelDecoration))
```

Do you still need this part? The calculation of `ci.HeaderMatchesProperty` already take into account whether or not the experimental feature is enabled.

Reply:
You are correct, removed.